src/core/sys/linux/elf.d

```diff
@@ -5,7 +5,15 @@
  */
 module core.sys.linux.elf;
 
-version (linux):
+// This interface is really a CRuntime_Glibc header, but as the bindings are
+// in a platform-specific module, that would mean this 2500+ line file would
+// have to be copied for each platform that is exclusively glibc.
+// Selectively permit other platforms to import the symbols in this module.
+
+version (linux) version = LinuxOrCompatible;
+version (Hurd)  version = LinuxOrCompatible;
+
+version (LinuxOrCompatible):
```

Comment:
This is the only weird case that needs considering.

Reply:
ELF for Linux or Hurd, is exactly the same according to the OS identifier.

Reply:
yes, they are the same